src/test/kotlin/com/exactpro/th2/http/client/ResponseTest.kt

```diff
@@ -0,0 +1,107 @@
+package com.exactpro.th2.http.client
```

Comment:
I don't see a need to involve network here. We should rather test that Th2RawHttpRequest behaves correctly e.g. its with* method returns an instance which retains parent event id